exercises/bowling/bowling_test.py

```diff
@@ -1,258 +1,189 @@
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1
```

Comment:
For consistency with the other exercises, could you please move this line like so:
```Python
from bowling import BowlingGame


# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1

class BowlingTests(unittest.TestCase):
...
```
Please note the number of blank lines before and after.

Reply:
done!